gifted_build/postgres_manage.js

```diff
@@ -0,0 +1,97 @@
+            table.string('size_and_fit', 511)
+            table.string('materials_and_care', 511)
+            table.decimal('price', 6, 3) //6 digits precision (total number of digits), 5 digits scale (# of digits to right of decimal point)
+            // The product description displayed in bold to the right of the product image
+            table.string('description', 511)
+            // The "conventional" method is to use a linking table
+            table.json('colors')
+        })
+
+        console.log(`Creating ${product_images_table} table`)
+        await knex.schema.createTable(product_images_table, table => {
+            table.string('SKU', 255).references(`${products_table}.SKU`)
+            table.binary('image')
+        })
+}
+
+// Inserts sample data into the tables
+async function populateDB() {
+    console.log(`Populating ${products_table} table`)
+    await knex(products_table).insert([
+        {
+            SKU: '36523641234523',
+            title: "Unisex Combined Logo & Script Tee",
+            product_info: `A large 'GG' logo printed on the front with our customer-favorite script logo underneath.`,
+            size_and_fit: `Our model is 5'10" (178 cm), 165 pounds (75 kg), and is wearing a size medium.\nTear away tag.\nClassic fit.`,
+            materials_and_care: `50/50 cotton/polyester.\nMachine washable, cold, with like colors.`,
+            price: 22.00,
+            description: `A portion of this sale goes towards Special Olympics New York. Thank you for continuing to support this great foundation. You can read more about our partnership by visiting the "Our Partners" section of our website.\nClassic, cool, comfortable. The new style in streetwear starts with Gifted.\nWith a fresh yet simple style, Gifted provides a slick, high quality, unisex tee.`,
+            colors: JSON.stringify(['black'])
+        },
+        {
+            SKU: '21354654',
+            title: `Unisex 'Everyone Is' Back Reflection Tee`,
+            product_info: `'Everyone is' slogan back fade.\nOur classic 'GG' logo on the front left chest with our iconic slogan 'Everyone is' printed repeatedly and reflected on your favorite short sleeve tee.`,
+            size_and_fit: `Our model is 5'9" (175 cm), 143 pounds (65 kg) and is wearing a size medium.\nTear away tag.\nClassic fit.`,
+            materials_and_care: `50/50 cotton/polyester.\nMachine washable, cold, with like colors.`,
+            price: 22.00,
+            description: `A portion of this sale goes towards Special Olympics New York. Thank you for continuing to support this great foundation. You can read more about our partnership by visiting the "Our Partners" section of our website.\nClassic, cool, comfortable. The new style in streetwear starts with Gifted.\nWith a fresh yet simple style, Gifted provides a slick, high quality, unisex tee.`,
+            colors: JSON.stringify(['black', 'white'])
+        }
+    ])
+}
+
+(async () => {
```

Comment:
I don't think you need to async this btw

Reply:
All of the knex.js functions are async/return a `Promise`, so I think I have to use `async`/`await` or `.then`. While this doesn't seem too great, the lower-level node-postgres/`pg` library, forces the user to use callbacks, which imo, is even worse.

I'll see if there's a way to make the code more readable.